snippet-extractor-output/snippets.yaml

```diff
@@ -13357,6 +13357,50 @@
         console.log("Search term: " + result.searchTerm + " => Count: " + length);
       });
     });
+'Word.Body#type:member':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/50-document/manage-footnotes.yaml
+
+    // Get the referenced note's type, which is footnote.
+
+    await Word.run(async (context) => {
+      const footnotes = context.document.body.footnotes;
+      footnotes.load("items");
+      await context.sync();
+
+      const referenceNumber = $("#input-reference").val();
+      const mark = (referenceNumber as number) - 1;
+      const item = footnotes.items[mark];
+      console.log(`Note type of footnote ${referenceNumber}: ${item.type}`);
+
+      item.body.load("type");
+      await context.sync();
+
+      console.log(`Body type of note: ${item.body.type}`);
+    });
+'Word.BodyType:enum':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/50-document/manage-footnotes.yaml
+
+    // Get the referenced note's type, which is footnote.
+
+    await Word.run(async (context) => {
+      const footnotes = context.document.body.footnotes;
+      footnotes.load("items");
+      await context.sync();
+
+      const referenceNumber = $("#input-reference").val();
+      const mark = (referenceNumber as number) - 1;
+      const item = footnotes.items[mark];
+      console.log(`Note type of footnote ${referenceNumber}: ${item.type}`);
+
+      item.body.load("type");
+      await context.sync();
+
+      console.log(`Body type of note: ${item.body.type}`);
```

Comment:
For me, the console.log on line 13397 in this file and the console.log on 13402 return the same thing: "Footnote"

<img width="333" alt="image" src="https://user-images.githubusercontent.com/7977793/196541668-17f5c5db-9283-4c26-b65f-099c7d13720e.png">

Should the `item.type` and the `item.body.type` be different?